docs/Integrate/Indexers/TheGraph.md

```diff
@@ -0,0 +1,259 @@
+---
+title: "The Graph"
+sidebar_position: 1
+---
+
```

Comment:
`docs/Integrate/Indexers/TheGraph.md` -> `docs/integrate/indexers/the-graph.md`

Reply:
@pustovalov Thanks for the update ... please feel free to update this PR to make it clean ... Will continue updating the missing content from my end.